tests/runner/test_section.py

```diff
@@ -273,3 +273,30 @@ def test_sectionizer_inserts_empty_section_if_loader_pattern_mismatches(
     assert s[0].pattern == patterns[0]
     assert len(s[1]) == 1
     assert s[1].pattern == patterns[1]
+
+@pytest.mark.parametrize("padding", [False, True])
+def test_sectionizer_sets_padding_property(mocker: MockerFixture, padding: bool):
+    p = Pipeline(
+        loader=make_test_loader(mocker, pattern=Pattern.projection),
+        methods=[make_test_method(mocker, pattern=Pattern.projection, padding=padding)],
+    )
+    s = sectionize(p)
+
+    assert s[-1].padding == padding
```

Comment:
Is this comparing two `bool` values? If so, it might be preferred to use `is`:
```suggestion
    assert s[-1].padding is padding
```